src/utilities/F90_HYPRE_general.c

```diff
@@ -26,6 +26,30 @@ hypre_F90_IFACE(hypre_finalize, HYPRE_FINALIZE)
    *ierr = (hypre_F90_Int) HYPRE_Finalize();
 }
 
+void
+hypre_F90_IFACE(hypre_setmemorylocation, HYPRE_SETMEMORYLOCATION)
```

Comment:
Hi @liruipeng .  It looks like these are not yet used in our examples, correct?  Would it be easy to add them to one of the example drivers (at some point, not necessarily now)?  Thanks!

Reply:
Hi, @rfalgout . They are sued in ex5f https://github.com/hypre-space/hypre/blob/f51e1663ec33dc545af0a1d7805e9616a4751018/src/examples/ex5f.f#L109

Reply:
Ahh.  Somehow I missed that.  Thanks!